php/Makefile

```diff
@@ -0,0 +1,15 @@
+default:
+	echo "Nothing to do"
+
+install:
+	bin/composer.phar install
+
+update:
+	git pull
+	bin/composer.phar update
+
+start:
+	php -S localhost:8888 -t src/
+
+test:
+	./vendor/bin/phpunit tests
```

Comment:
can we add all these as `PHONY` targets please?

Reply:
Well I learned something new today. Cool!